core/src/main/resources/error/error-classes.json

```diff
@@ -1035,6 +1040,11 @@
       "The expected format is ByteString, but was <unsupported> (<class>)."
     ]
   },
+  "INVALID_COLUMN_NAME" : {
```

Comment:
This seems a Hive specific error. Could you add the word `HIVE` to the error class name like `INVALID_HIVE_COLUMN_NAME`